packages/file-asset-apis/src/s3/client-types/client-response.ts

```diff
@@ -4,13 +4,16 @@ export type S3Error = _Error;
 export type S3ErrorExceptions = S3ServiceException;
 
 export {
-    GetObjectOutput,
+    BucketAlreadyExists,
+    BucketAlreadyOwnedByYou,
+    NoSuchKey,
     ListObjectsV2Output,
-    PutObjectOutput,
     PutObjectTaggingOutput,
     DeleteObjectOutput,
     DeleteObjectsOutput,
```

Comment:
what is the difference between `GetObjectOutput` and `GetObjectCommandOutput`. If its important, why are not the rest of the `Output` types converted to their `CommandOutput` variants

Reply:
The `GetObjectOutput` type was missing from the added API types, which would result in a TypeScript error. This issue became a problem because we rely on the responses generated by the `transformToString` method in our use cases. The correct types are included in `GetObjectCommandOutput`.

Reference to resolved type errors:
https://github.com/terascope/file-assets/pull/955/files#diff-835e064de8a57fb59a4ab94220877a281113fbdce149878d23803b0fc2263828L48
https://github.com/terascope/file-assets/pull/955/files#diff-1c8c4efed4a3f6cf8beb81020f1c8f5b079549b0cdca137d0ab91d606b9a5d4eL97

Regarding the decision not to modify other types to match, it was primarily because I hadn't encountered any good reasons to do so. My intention was to make only the necessary changes and avoid unnecessary modifications where possible. However, I do believe that updating all output types to their `command` counterparts would be appropriate, as the default return types for all of them are `commandOutput` types. (I learned this by using the aws-sdk client without file-asset-apis)